app/models/region.rb

```diff
@@ -38,6 +48,19 @@ def short_uuid
     SecureRandom.uuid[0..7]
   end
 
+  def assigned_patients
+    Patient.where(assigned_facility: facilities)
+  end
+
+  def facilities
+    if facility_region?
+      Facility.where(id: source_id)
+    else
+      ids = facility_regions.pluck(:source_id)
```

Comment:
nitpick: Maybe call this `source_ids` for consistency and no confusion?

Reply:
👍